Theme (dark mode) toggle

jasonvarga opened the pull request:
Gives the option to match the system preference, or explicitly use light or dark themes.

Konafets commented:
Hey, 

thanks for adding the toggle. Just tried it with Chrome and the toggle switches to light mode, but the setting is lost after reloading the page.

jasonvarga commented:
I renamed some variables at the last second and it broke that without noticing. Derp. I've fixed this in 35f6853afbf7be3993d5800895825bca3f44b2e6, sorry about that.
